pkg/service/client.go

```diff
@@ -0,0 +1,263 @@
+package service
+
+import (
+	"context"
+	"fmt"
+
+	"capnproto.org/go/capnp/v3"
+	"github.com/libp2p/go-libp2p-core/peer"
```

Comment:
Found it!  This import should be changed to: `"github.com/libp2p/go-libp2p/core/peer"`